bridged_interactor/init.yml

```diff
@@ -1,6 +1,6 @@
 unbuffered: True
 archive: seed2.zip
-interactive: {files: interactor.cpp}
+interactive: {files: interactor.cpp, return_code: testlib}
```

Comment:
Ahhhh, this is what you meant... I think we might want to call this `type` or something, so that we don't end up in the future having to convert `return_code` -> `type` when something else is added. Code should be minimalist on the features it implements at first, but configs ought to be general for future extensibility.